README.md

```diff
@@ -30,10 +32,20 @@ config :sentry,
 ```
 
 ### Capture Exceptions
+
+Sometimes you want to capture specific exceptions, to do so use the `Sentry.capture_exception/3`.
+
 ```elixir
-Sentry.capture_exception(my_exception)
+try do
+  ThisWillError.reall()
+rescue
+  my_exception ->
+    Sentry.capture_exception(my_exception)
```

Comment:
oops, I got it wrong.  It would be `Sentry.capture_exception(my_exception, [stacktrace: System.stacktrace(), extra: %{extra: information}])`

Stacktraces are not super intuitive, so I'm not sure if we should refer to other documentation on them, or write some of our own.